src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/CaseData.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.ecm.common.model.reports.hearingstojudgments;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import uk.gov.hmcts.ecm.common.model.ccd.items.HearingTypeItem;
+import uk.gov.hmcts.ecm.common.model.ccd.items.JudgementTypeItem;
+
+import java.util.List;
+
+@Data
+public class CaseData {
+    @JsonProperty("ethosCaseReference")
+    private String ethosCaseReference;
+
+    @JsonProperty("hearingCollection")
+    private List<HearingTypeItem> hearingCollection;
+
+    @JsonProperty("judgementCollection")
+    private List<JudgementTypeItem> judgementCollection;
+
+    @JsonProperty("managingOffice")
+    private String managingOffice;
```

Comment:
May be this class name can be changed to something report specific to clearly differentiate the generic CaseData?

Reply:
renamed hearings to judgments case data object to be more distinct